modin/core/execution/ray/common/utils.py

```diff
@@ -219,6 +219,17 @@ def initialize_ray(
             if not GPU_MANAGERS:
                 for i in range(GpuCount.get()):
                     GPU_MANAGERS.append(GPUManager.remote(i))
+    else:
+        ray_obj_store_mem = ray.available_resources()["object_store_memory"]
+        virtual_memory = psutil.virtual_memory().total
```

Comment:
Actually, in this context you might want to use available since there could be other processes running that are consuming memory?

Reply:
Free vs. available memory in Linux: https://haydenjames.io/free-vs-available-memory-in-linux/